object-store-client-common/src/main/scala/uk/gov/hmrc/objectstore/client/ObjectStoreClient.scala

```diff
@@ -35,37 +35,23 @@ class ObjectStoreClient[F[_], REQ_BODY, RES, RES_BODY](
 
   private val url = s"${config.baseUrl}/object-store"
 
-  def putObject[CONTENT](
-    path    : Path.File,
-    content : CONTENT
-  )(implicit
-    w: ObjectStoreContentWrite[F, CONTENT, REQ_BODY]
-  ): F[Unit] =
-    putObject[CONTENT](path, content, config.owner)
-
   def putObject[CONTENT](
     path       : Path.File,
     content    : CONTENT,
-    owner      : String
+    contentType: Option[String] = None,
```

Comment:
LGTM. Can we document what will be the default content-type so the users of this lib know what to expect?

Reply:
I mean Scaladoc